### PR TITLE
common events paste

### PR DESCRIPTION
**Pull request by neotyk**
Porting `:paste` event to common events, only locally for now.
#1413

### CODE REVIEW

File: src/cljs/athens/events.cljs [hunk 1]
```diff
@@ -1607,6 +1609,21 @@
                                         embed-id (str "-embed-" embed-id)) n]))]})))
 
 
+#_(reg-event-fx
```
Comment (filipesilva) [inline src/cljs/athens/events.cljs]: I think this can be removed?

File: src/cljc/athens/common_events/resolver.cljc [hunk 2]
```diff
@@ -966,3 +967,115 @@
                       {:db/id [:block/uid uid] :block/string new-str}))
                   unlinked-refs)]
     tx-data))
+
+
+(defn text-to-blocks
```
Comment (filipesilva) [inline src/cljc/athens/common_events/resolver.cljc:972]: I'm a bit confused here, I see a lot of code was added but no code was removed. But the paste functionality already existed, so shouldn't its old code be removed in this PR?

File: src/cljc/athens/common_events.cljc [hunk 3]
```diff
@@ -524,14 +524,18 @@
...
+  [last-tx uid text start value]
   (let [event-id (gen-event-id)]
     {:event/id      event-id
      :event/last-tx last-tx
      :event/type    :datascript/paste
-     :event/args    {:uid  uid
-                     :text text}}))
+     :event/args    {:uid   uid
+                     :text  text
+                     :start start
```
Comment (filipesilva) [inline src/cljc/athens/common_events.cljc:537]: Will `:start` and `:value` be used later or something? They don't seem to be used now.

**Review by filipesilva — CHANGES_REQUESTED** [review]
Couple of comments, and also there's merge conflicts that need to be resolved before this is merged.

File: src/cljc/athens/common_events.cljc (hunk 3 above)
Comment (neotyk) [reply src/cljc/athens/common_events.cljc:567]: Resolver uses these values.

File: src/cljs/athens/events.cljs (hunk 1 above)
Comment (neotyk) [reply src/cljs/athens/events.cljs]: It's removed.

File: src/cljc/athens/common_events/resolver.cljc (hunk 2 above)
Comment (neotyk) [reply src/cljc/athens/common_events/resolver.cljc:1187]: Indeed, it wasn't removed from `events.cljs`

Comment (neotyk) [reply src/cljc/athens/common_events/resolver.cljc:1186]: Now it's gone from `events.cljs`.

**Review by filipesilva — APPROVED** [review]

**Pull request merged**